tools/shared/src/main/scala/org/scalajs/core/tools/linker/backend/emitter/Emitter.scala

```diff
@@ -165,6 +165,18 @@ final class Emitter private (config: CommonPhaseConfig,
       for (generatedClass <- generatedClasses)
         emitJSTrees(generatedClass.main)
 
+      // $L0 = new RuntimeLong(0, 0)
+      builder.addJSTree({
+        import TreeDSL._
+        implicit val pos = Position.NoPosition
+        js.Assign(
+            jsGen.envField("L0"),
+            js.Apply(
+                js.New(jsGen.encodeClassVar(LongImpl.RuntimeLongClass), Nil) DOT LongImpl.initFromParts,
+                List(js.IntLiteral(0), js.IntLiteral(0)))
+        )
+      })
```

Comment:
Consider moving the tree definition itself to a helper to keep this method cleaner. Also, consider updating the comment that static field definitions also includes `L0`.